### PR TITLE
New, 7.2 User | Sorting Search Results | LRDOCS-5931

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-5931, cc @lipusz

### CODE REVIEW

**Comment by rbohl** [comment]
@lipusz feel free to review that last commit and let me know if you want any last-minute updates :mag:

File: user/articles/120-search/04-sort-widget.markdown [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+**Field:** `createDate-`
+
+The `-` sign following the field name indicates that the order is _descending_.
+Choosing to sort with this will bring search results created most recently to
+the top of the list.
+
+**Label:** _Created (oldest first)_
+**Field:** `createDate+`
+
+The `+` sign following the field name indicates that the order is _descending_.
```
Comment (lipusz) [inline user/articles/120-search/04-sort-widget.markdown]: Change: "descending" appears twice: 

You can indicate the sort order by appending a + symbol (means, ascending) or a - symbol (means, descending) to the field name like createDate+

**Comment by lipusz** [comment]
@rbohl See above :-)

**Comment by rbohl** [comment]
:shipit: 
Ready, @sez11a

**Comment by sez11a** [comment]
Just started reviewing :)

**Pull request merged**